### ISSUE TITLE
Bug:Screen whiteout after record lock.

### ISSUE THREAD

**Issue by nogumaruo** [notum-cz/strapi-plugin-record-locking#2]
Hi!
Thanks for the nice plugin!

I have tried record locking with two admin users.
The user who edited later was indeed unable to edit, but that was because the screen was whiteed out and an error was output to the console

Since a record was created in "record-locking_open-entity" for an exclusivity check, there may be a problem on the "admin panel" source side.

```
│ Environment        │ development                                      │
│ Process PID        │ 70604                                            │
│ Version            │ 4.1.12 (node v16.14.0)                           │
│ Edition            │ Enterprise                         
```
<img width="1064" alt="Content_Manager" src="https://user-images.githubusercontent.com/98809021/170818306-3a9d6953-ab96-43af-8e74-7d5f0db24c77.png">
.

**Comment by nogumaruo** [comment]
@martincapek 
I am still getting whiteouts in 0.2.2, 0.2.3.

**Comment by martincapek** [comment]
Hi, sorry it should be fixed 0.2.4.

**Comment by nogumaruo** [comment]
I and other users are very eager for this plugin, so I'm rooting for improvements!
